tests/Tutor.Web.Tests/TestData/Scripts/x-learner-mastery.sql

```diff
@@ -14,62 +14,77 @@ INSERT INTO public."Learners"(
     "Id", "UserId", "Name", "Index")
 VALUES (-5, -5, 'Enrolled Student for Events', 'SU-5-2021');
 
+INSERT INTO public."Instructors"("Id", "UserId", "Name", "Surname") VALUES
+    (-51, -51, 'TestInstructor', 'TestInstructor');
+
+INSERT INTO public."Instructors"("Id", "UserId", "Name", "Surname") VALUES
+    (-52, -52, 'TestInstructor', 'TestInstructor');
+
 INSERT INTO public."LearnerGroups"(
 	"Id", "Name", "CourseId")
 VALUES (-1, 'Test Group', -1);
 INSERT INTO public."GroupMemberships"(
-	"Id", "LearnerId", "LearnerGroupId")
-	VALUES (-1, -1, -1);
+    "Id", "LearnerId", "InstructorId", "LearnerGroupId", "Role")
+    VALUES (-1, null, -51, -1, 0);
+INSERT INTO public."GroupMemberships"(
+	"Id", "LearnerId", "InstructorId", "LearnerGroupId", "Role")
+	VALUES (-2, -1, null, -1, 1);
 INSERT INTO public."GroupMemberships"(
-	"Id", "LearnerId", "LearnerGroupId")
-	VALUES (-2, -2, -1);
+    "Id", "LearnerId", "InstructorId", "LearnerGroupId", "Role")
+	VALUES (-3, -2, null, -1, 1);
```

Comment:
The learner role should be 0. It should go first in the enumeration as it will be more common.